tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/VersionConflict/AspNetVersionConflictTests.cs

```diff
@@ -0,0 +1,128 @@
+// <copyright file="AspNetVersionConflictTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETFRAMEWORK
+
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.VersionConflict
+{
+    [Collection("IisTests")]
+    public class AspNetVersionConflictTests : TestHelper, IClassFixture<IisFixture>
+    {
+        private readonly IisFixture _iisFixture;
+
+        public AspNetVersionConflictTests(IisFixture iisFixture, ITestOutputHelper output)
+            : base("AspNet.VersionConflict", @"test\test-applications\aspnet", output)
+        {
+            SetServiceVersion("1.0.0");
+
+            _iisFixture = iisFixture;
+            _iisFixture.ShutdownPath = "/home/shutdown";
+            // There is an issue in the TracingHttpModule that causes the parent trace to be locked, making the test useless
+            // This code is not run when IIS is in classic mode, so using that as a workaround
+            _iisFixture.TryStartIis(this, IisAppType.AspNetClassic);
+        }
+
+        [SkippableFact]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        [Trait("LoadFromGAC", "True")]
+        public async Task SubmitsTraces()
+        {
+            // 4 spans for the base request: aspnet.request / aspnet-mvc.request / Manual / http.request
+            // + 2 spans for the outgoing request: aspnet.request / aspnet-mvc.request
+            const int expectedSpans = 6;
+
+            var spans = await GetWebServerSpans("/home/sendrequest", _iisFixture.Agent, _iisFixture.HttpPort, System.Net.HttpStatusCode.OK, expectedSpans, filterServerSpans: false);
+
+            spans.Should().HaveCount(expectedSpans);
+
+            foreach (var span in spans)
+            {
+                Output.WriteLine($"{span.Name} - {span.TraceId} - {span.SpanId} - {span.ParentId} - {span.Resource}");
+            }
```

Comment:
[nit] I'd move the assertion below the log, so if it fails you can see why

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2020/commits/e1b26a25a9095a2a389a61a409e792c1b84a20b4